src/ui/components/Transcript/Transcript.tsx

```diff
@@ -24,7 +24,7 @@ export default function Transcript() {
       clonedComments.push(pendingComment.comment);
     }
 
-    const sortedComments = sortBy(clonedComments, ["point", "createdAt"]);
+    const sortedComments = sortBy(clonedComments, [c => BigInt(c.point), "createdAt"]);
```

Comment:
I'm surprised this is necessary. I thought JavaScript's normal string sorting would properly handle numeric string values in cases like this.

Taking the points from one of the Replays you mentioned, it looks like this is just a difference in behavior between Lodash `sortBy` and native JavaScript `sort`: https://codesandbox.io/s/boring-sound-lic91z?file=/src/index.js

TIL.

Reply:
Strings are always sorted lexicographically, in the codesandbox you use `(a, b) => a.point - b.point` as the sort function, which will coerce the strings to numbers, `sortBy` will not do that type coercion.